hops/featurestore.py

```diff
@@ -1813,3 +1813,21 @@ def get_training_dataset_statistics(training_dataset_name, featurestore=None, tr
     except:
         core._get_featurestore_metadata(featurestore, update_cache=True)
         return core._do_get_training_dataset_statistics(training_dataset_name, featurestore, training_dataset_version)
+
+def connect(host, project_name, port = 8181):
```

Comment:
@steffengr it's better the default port is `443`

Reply:
Indeed. I will change it.

Reply:
It appears like it assumes the same region as the SageMaker instance is running in by default which seems like a reasonable requirement for now and  avoid specifying and passing the values around everywhere. I'll simply remove it for now.